packages/xo-server/src/models/remote.js

```diff
@@ -15,6 +15,14 @@ export class Remotes extends Collection {
     const remotes = await super.get(properties)
     forEach(remotes, remote => {
       remote.enabled = remote.enabled === 'true'
+
+      let benchmarks
+      try {
+        benchmarks = JSON.parse(remote.benchmarks)
+      } catch (error) {
```

Comment:
Better check if `benchmarks` is defined.

Reply:
I add a `try/catch` to avoid issue with `JSON.parse`, but its ok for me :)